<ui version="4.0" >
 <class>drawerSettings</class>
 <widget class="QWidget" name="drawerSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>1</number>
   </property>
   <property name="leftMargin" >
    <number>2</number>
   </property>
   <property name="topMargin" >
    <number>1</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <property name="bottomMargin" >
    <number>1</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="horizontalSpacing" >
      <number>0</number>
     </property>
     <property name="verticalSpacing" >
      <number>1</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="font" >
        <font>
         <family>Tahoma</family>
        </font>
       </property>
       <property name="text" >
        <string>Фоновый цвет</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>51</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="ambientEdit" >
       <property name="autoFillBackground" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="font" >
        <font>
         <family>Tahoma</family>
        </font>
       </property>
       <property name="text" >
        <string>Рассеянный цвет</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>51</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2" >
      <widget class="QPushButton" name="diffuseEdit" >
       <property name="autoFillBackground" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="font" >
        <font>
         <family>Tahoma</family>
        </font>
       </property>
       <property name="text" >
        <string>Отраженный цвет</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>51</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2" >
      <widget class="QPushButton" name="specularEdit" >
       <property name="autoFillBackground" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="font" >
      <font>
       <family>Tahoma</family>
      </font>
     </property>
     <property name="title" >
      <string>Расположение источника</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>1</number>
      </property>
      <property name="topMargin" >
       <number>1</number>
      </property>
      <property name="rightMargin" >
       <number>1</number>
      </property>
      <property name="bottomMargin" >
       <number>1</number>
      </property>
      <property name="horizontalSpacing" >
       <number>1</number>
      </property>
      <property name="verticalSpacing" >
       <number>1</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="font" >
         <font>
          <family>Tahoma</family>
         </font>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text" >
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" >
       <widget class="QLineEdit" name="xEdit" >
        <property name="maximumSize" >
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="font" >
         <font>
          <family>Tahoma</family>
         </font>
        </property>
        <property name="text" >
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" >
       <widget class="QLineEdit" name="yEdit" >
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>35</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="font" >
         <font>
          <family>Tahoma</family>
         </font>
        </property>
        <property name="text" >
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" >
       <widget class="QLineEdit" name="zEdit" >
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>35</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
